content/post/roadmap.md

```diff
@@ -0,0 +1,60 @@
+
+You may end up tying your work back to, say, 2 of your organization's 3 strategic goals for the year. That's not in and of itself bad, but should be called out. This exercise reduces blind spots and forces you to think about how you can support all aspects of the mission.
+
+### Set up the scaffolding
+
+How do you start? After outlining our Data team mission and values (distinct from the broader company) I like to make explicit what areas of focus we're responsible for. For us, that's Data infrastructure, BI, analytics and strategic projects, Data Science and Clinical Data products. This helps to compartmentalize projects, especially those that are cross functional.
+
+You may find it more natural to split your road map by the various teams you support, e.g. Marketing, Product, Sales, etc... I prefer the former because it captures both technical differences that are relevant to the team (e.g. data analysts will skew heavily towards BI, data engineers towards data infrastructure, etc...) and "Analytics and Strategic projects" encapsulate projects that are cross functional (e.g. Marketing, Sales, Finance).
+
+### Sync with functional leaders 
+
+After putting down some projects that fit into the relevant areas of focus, set up time with key functional leaders and get their input. Hopefully you sequence this exercise at a time in which they are also thinking about their goals for the year. Have them help you fill in blind spots and loosely prioritize their projects. You may know of 4 Operations projects that are critical for 2019, but the COO may have a different idea on what the sequencing should be. Don't get bogged down by timing at this point, but getting the order correct is very helpful.
+
+Functional leaders will also be helpful in identifying cross team dependencies. You may have grand plans for implementing [Airflow](https://airflow.apache.org/), but if you lack Data Engineering resources, you may have to rethink your plan. Similarly you may have an idea for a great data-driven product feature, but if Product isn't prioritizing that in their roadmap, it may drop to the bottom of your priority, even though you think it's a valuable feature.
+
+### Put stakes in the ground 
+
+Once you're confident that you have covered the most important projects for the year, write down estimated timing for completion and key stakeholders impacted. For example, under "Analytics" you may have the bullet `Establish a multi-touch attribution model // Q2 // Marketing`.
+
+After you do this across the board, you'll have a good sense of how projects are being sequenced, if you're front-loading projects (too many in Q1/Q2), and which functions aren't being supported (or are being overly represented). This functions as another sense check for how you're orienting your year and helps to communicate and set expectations across the organization. 
```

Comment:
The re-use of "functions" in two different sentences back-to-back here is a little confusing